__tests__/logs/json-server.log

```diff
@@ -0,0 +1 @@
+
```

Comment:
When running this locally I'm seeing this file change (as expected). What is unexpected is that this is checked in. Should we figure out a way to ignore this log?

Reply:
this file should not be committed.  added to .gitignore and removed.